tests/test_grpc_server.py

```diff
@@ -348,6 +348,28 @@ def test_resumable_query_completed(self):
         self.assertEqual(blob.bucket, "projects/_/buckets/bucket-name")
         self.assertEqual(blob.size, len(media))
 
+    def test_list_objects(self):
+        names = ["a/test-0", "a/test-1", "a/b/test-0", "a/b/test-1", "c/test-0"]
+        media = b"The quick brown fox jumps over the lazy dog"
+        for name in names:
+            request = testbench.common.FakeRequest(
+                args={"name": name}, data=media, headers={}, environ={}
+            )
+            blob, _ = gcs.object.Object.init_media(request, self.bucket.metadata)
+            self.db.insert_object(request, "bucket-name", blob, None)
+        context = unittest.mock.Mock()
+        response = self.grpc.ListObjects(
+            storage_pb2.ListObjectsRequest(
+                parent="projects/_/buckets/bucket-name",
+                prefix="a/",
+                delimiter="/",
```

Comment:
Add a test for start_offset, end_offset, and include_trailing_delimiter

Reply:
Done.